FAQ.md

```diff
@@ -83,37 +83,37 @@ looking at some of these things in the future and provide guidance around them.
 
 ## Development
 
-### Is the UI-Kit available with my content management system or JavaScript framework?
-The UI-Kit is built to be system agnostic. This means that it can work with any system, development environment, build tools or stack.
+### Is the components available with my content management system or JavaScript framework?
+The components is built to be system agnostic. This means that it can work with any system, development environment, build tools or stack.
```

Comment:
```suggestion
The components are built to be system agnostic. This means that it can work with any system, development environment, build tools or stack.
```